board-games/test/board-game-test.js

```diff
@@ -0,0 +1,70 @@
+var assert = require('chai').assert;
+var Game = require('../src/board-game.js');
+
+describe('Board Game', function() {
```

Comment:
Might be most consistent to just call this "Game"!

Reply:
changed